doc/whats_new/v0.20.rst

```diff
@@ -363,6 +363,13 @@ Neighbors
 
 Feature Extraction
 
+- Deprecate ``stop_words='english'`` parameter in
```

Comment:
This should be in `API changes summary` section

Reply:
@qinhanmin2014  Oh, will move it there. Can you please let me know which subsection in `API changes summary` or shall I make a new subsection named Feature Extraction in `API changes summary` section? Thanks!

Reply:
a new subsection please

Reply:
Done. :+1: